.travis.yml

```diff
@@ -14,33 +13,47 @@ matrix:
   include:
     - os: linux
       dist: bionic
+      python: '3.6'
       addons:
         apt:
           packages:
-          # Will still load inkscape 0.92.x, hence use manual app image, see below
-          #- inkscape
           - python3-gi
           - python3-gi-cairo
           - gir1.2-gtk-3.0
           - texlive-latex-base
+          - texlive-fonts-recommended
+          - texlive-lang-cyrillic
+          - cm-super
+          - libxml2-utils
+
+    - os: linux
+      dist: focal
+      python: '3.8'
+      addons:
+        apt:
+          packages:
+          - python3-gi
+          - python3-gi-cairo
+          - gir1.2-gtk-3.0
+          - texlive-latex-base
+          - texlive-lang-cyrillic
```

Comment:
I don't think it would test anything _in textext_. The #240 is about environment configuration.

Reply:
I think at least the "flow" of the characters is tested if the configuration is correct.